mergify_engine/actions/comment.py

```diff
@@ -27,7 +27,9 @@
 
 class CommentAction(actions.Action):
     validator = {
-        voluptuous.Required("message"): types.Jinja2,
+        voluptuous.Required("message", default=None): voluptuous.Any(
```

Comment:
The code in `run` doesn't seems to handle the `None` case.

Reply:
#1988 fix this issue